### PR TITLE
Added Azure Postgresql migration extension public preview

### PR DESCRIPTION
**Pull request by sagangwar**

### CODE REVIEW

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
You need to add this to the insiders gallery as well

File: extensionsGallery.json [hunk 1]
```diff
@@ -4893,6 +4893,71 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "97",
+					"extensionName": "postgresql-migration",
+					"displayName": "Azure PostgreSQL migration",
+					"shortDescription": "Provides a tool to assess and migrate PostgreSQL database(s) to Azure database for PostgreSQL",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4901]: (s) is difficult to translate so typically we prefer to just use the plural form

```suggestion
					"shortDescription": "Provides a tool to assess and migrate PostgreSQL databases to Azure database for PostgreSQL",
```

**Comment by sagangwar** [comment]
> d to add this to the insiders gallery as well

Done

**Comment by sagangwar** [comment]
> You need to add this to the insiders gallery as well

**Comment by sagangwar** [comment]
> You need to add this to the insiders gallery as well

**Comment by sagangwar** [comment]
> @sagangwar Looks like the icon file isn't correct
> 
> `Error: /home/runner/work/azuredatastudio/azuredatastudio/extensionsGallery.json - postgresql-migration - Error fetching Microsoft.VisualStudio.Services.Icons.Default with URL https://dsct.blob.core.windows.net/extensions/postgresql-migration/2.0.3/postgreSQLMigrationIcon.png. HTTPError: Response code 404 (The specified blob does not exist.)`

We have corrected it and Build is succeeded now, please approve it now.

**Comment by sagangwar** [comment]
> **MSFT Gallery Maintainer Only** This is a reminder to update the Extensions Gallery list at https://github.com/Microsoft/azuredatastudio/wiki/List-of-Extensions once this PR has been merged in. Instructions for doing this can be found [here](https://github.com/microsoft/azuredatastudio-docs/blob/main/Guides/UpdateExtensionGallery.md#guide-for-extension-release-dri).

Okay

**Review by Charles-Gagnon — APPROVED** [review]

**Pull request merged**